server/etcdserver/api/snap/snapshotter.go

```diff
@@ -25,6 +25,8 @@ import (
 	"strings"
 	"time"
 
+	"go.etcd.io/etcd/client/pkg/v3/verify"
+
```

Comment:
Please remove this blank line and reorder the import entries using `gofmt -w snapshotter.go`

Reply:
You only need to remove the blank line.

Reply:
Have updated.